facebook-commerce-events-tracker.php

```diff
@@ -19,10 +19,10 @@ class WC_Facebookcommerce_EventsTracker {
 
   public function __construct($user_info) {
     $this->pixel = new WC_Facebookcommerce_Pixel($user_info);
-    self::apply_filters();
+    add_action('wp_head', array($this, 'apply_filters'), 9);
```

Comment:
Why it's 9?

Reply:
Because default priority is 10 and it will guarantee that this action will be fired before others.

Reply:
10 should be good enough and we can just skip this param. It's a good idea to move the rest of [actions](https://github.com/facebookincubator/facebook-for-woocommerce/blob/master/facebook-commerce.php#L263-L283) here. Would you like to change this as well?

Reply:
Hi @mister ,

If you dont want to change the action setup, we can change it on our side. Can you update the priority then i will accept. Since we are using ShipIt to automatically sync with github, I can't help change your code on my side. Thanks for your pull request!

Reply:
Hi @mengyingdu ,

I'll try to update it today or tomorrow. Not a problem, was busy few past days.